cfpb/cfpb-chart-builder
# Bug fix: import highcharts.less locally instead of as external request

Fixes bug where highcharts.less default styles were imported as an external http request. This caused an increase in page load time on the latest version of chart-builder. This PR imports the same code from a local less file instead.

## Additions

- `highcharts.less` - contains the default highcharts styling for charts in styled mode.

## Changes

- `cfpb-chart-builder.less` now imports the local highcharts.less instead of the external URL.

## Testing

- checkout branch
- run `gulp build` and `gulp watch` 
- confirm demo styles match screenshots from https://github.com/cfpb/cfgov-refresh/pull/3173

## Review

- @mistergone OR @anselmbradford OR @contolini 

[Preview this PR without the whitespace changes](?w=0)



## Checklist
